lib/AST/ASTImporter.cpp

```diff
@@ -1136,8 +1175,8 @@ bool ASTNodeImporter::ImportDeclParts(NamedDecl *D, DeclContext *&DC,
   }
   
   // Import the name of this declaration.
-  Name = Importer.Import(D->getDeclName());
-  if (D->getDeclName() && !Name)
+  if (auto Err = ImportOrError(Name, D->getDeclName()))
+    // FIXME: handle error
```

Comment:
We could just return the `Err` here, which means ImportDeclParts should change its return type from `bool` to `Error`.